profiles/thurloe-configs/terraform/thurloe/variables.tf.ctmpl

```diff
@@ -132,3 +132,7 @@ variable "bucket_tag" {
   description = "to use in config rendering"
   default = "{{if env "BUCKET_TAG"}}{{env "BUCKET_TAG"}}{{else}}{{env "ENVIRONMENT"}}{{end}}"
 }
+
+variable "consul_log_level" {
+  default = "{{if env "LOG_LEVEL"}}{{env "LOG_LEVEL"}}{{else}}err{{end}}"
```

Comment:
should this be `CONSUL_LOG_LEVEL`?

Reply:
seems like it but no. 

The "render terraform templates" step uses the `render_templates.sh` script from dsde-toolbox, where the log level is called `LOG_LEVEL`. It comes from https://github.com/broadinstitute/dsde-toolbox/blob/master/render-templates.sh#L47 

The reason that dsp-k8s-deploy uses `render-templates.sh` instead of calling `consul-template` directly is that `render-templates` already knows how to handle a whole directory of files.

Reply:
ok!